dependencymagnet/doc.go

```diff
@@ -1,3 +1,4 @@
+//go:build tools
```

Comment:
I think this line is being added because you are running the generation locally with the latest go version. Run the `make generate-with-container` target to build it with the correct versions.

Reply:
I ran the `generate-with-container` but `verify` after it. 
So I think it's being regenerated.

Reply:
@sherine-k with go 1.17 the tags have changed a bit, so unless that's changed in [go.mod which still is on 1.16](https://github.com/openshift/api/blob/master/go.mod#L3), please take 1.16 or run it through the container